lib/contentdm_translator.rb

```diff
@@ -136,6 +136,25 @@ def self.fts_field_for_collection(collection)
     return error, fts
   end
 
+  def self.export_work_to_cdm_with_retry(work, username, password, license)
+    max_delay = 3600
+    delay = 300
+
+    begin
+      ContentdmTranslator.export_work_to_cdm(work, username, password, license)
+    rescue Net::ReadTimeout => e
+      print "Net::ReadTimeout: Retrying in #{delay} seconds... (#{e.message})"
+
+      sleep(delay)
+
+      delay *= 2
```

Comment:
This will only do a back-off four times, with a max time-out of 6 minutes.  We're seeing some longer outages lately, so I'd prefer to raise the back-off to somewhere around 4-6 hours.